tests/test_agentspec_smoke.py

```
@@ -0,0 +1,69 @@
+import sys
+
+import pytest
+
+# Prepend local src to sys.path to ensure local NAT is used
+SRC_DIR = os.path.join(os.path.dirname(os.path.dirname(__file__)), "src")
+if SRC_DIR not in sys.path:
+    sys.path.insert(0, SRC_DIR)
+
+
+@pytest.mark.asyncio
+async def test_agentspec_adapter_smoke(monkeypatch):
```

Ruff (0.14.13)

Unused function argument: `monkeypatch`

(ARG001)